src/main/resources/avro/references.avdl

```diff
@@ -31,65 +31,43 @@ field. New client and server implementations should prefer the "graph" mode.
 record Reference {
 
   /**
-  The `Segment` describing this `Reference`'s sequence in the `ReferenceSet`'s
+  The `Segment` describing this `Reference`'s `Sequence` in the `ReferenceSet`'s
   sequence graph.
 
-  The `Segment` will generally cover the entirety of the sequence, but may be
-  shorter if only a subrange of a sequence is to be included in the
+  The `Segment` will generally cover the entirety of the `Sequence`, but may be
```

Comment:
This is probably just me being dense but, is there a good reason besides this for continuing to hold on to the `Segment` as an entity, now that `Sequence` is one? And even in this case, couldn't a `Reference` simply include a `Sequence` ID together with `start` and `length` fields to denote the relevant subrange?